### PR TITLE
Support oem and install_inband fields

### PR DESCRIPTION
**Pull request by ofaurax**
FS-1439 and FS-1468

### CODE REVIEW

**Review by DoctorVin — DISMISSED** [review]
Consider renaming the files that are named `serverserviceX.go` to `fleetdb`.

File: internal/inventory/mocks/serverservice.go
Comment (coffeefreak101) [inline internal/inventory/mocks/serverservice.go:1]: Should probably rename this file to fleetdb.go?

File: internal/inventory/serverservice.go
Comment (coffeefreak101) [inline internal/inventory/serverservice.go:1]: Should probably rename this file to fleetdb.go?

File: internal/inventory/serverservice_test.go
Comment (coffeefreak101) [inline internal/inventory/serverservice_test.go:1]: Should probably rename this file to fleetdb_test.go?

File: internal/inventory/mocks/serverservice.go
Comment (ofaurax) [reply internal/inventory/mocks/serverservice.go:1]: Done for the 3 files

**Review by DoctorVin — APPROVED** [review]

**Pull request merged**